components/main.css

```diff
@@ -0,0 +1,259 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+:root{
+    font-family: 'Poppins', sans-serif;;
+}
+.logo a{
+    display: flex;    
+    padding: 2rem 3rem 1rem 6rem;
+ }
+.logo-text{
+    font-size: 1.7rem;
+    font-family: 'Roboto Slab', serif;
+    padding-left:.5rem ;
+    font-weight: 600;
+    color: var(--black);
+    cursor: pointer;
+
+}
+.text{
+    font-size: 1.7rem;
+    color:var(--main-color);
+    font-family: 'Roboto Slab', serif;
+    cursor: pointer;
+   }
+.logo-img{
+    height: 30px;
+    width: 30px;
+}
+.container{
+    display: flex;
+    max-width: 81.25rem;
+}
+.sidebar-wrapper{
+    width: 20rem;
+    height: 100%;
+    overflow-y: auto;
+    background-color: var(--grey);
+    position: fixed;
+    font-size: 1rem;
+    z-index: 5;
+    box-shadow: rgba(34, 33, 33, 0.35) 2px 5px 15px;
```

Comment:
you should take variables for colors.

Reply:
you can make 1 css file and add all repeated colors and classes and then import it into project css file.